src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -217,7 +217,7 @@ std::string KeyboardShortcuts::GetShortcutString(int32_t shortcut) const
 
 ScreenCoordsXY KeyboardShortcuts::GetKeyboardMapScroll(const uint8_t* keysState) const
 {
-    ScreenCoordsXY screenCoords;
+    ScreenCoordsXY screenCoords{};
```

Comment:
I know you did it with the best of intentions, but this doesn't relate to the task at hand, so can you please revert?
```suggestion
    ScreenCoordsXY screenCoords;
```